adopters.geojson

```diff
@@ -81,5 +81,6 @@
     {"type": "Feature", "properties": {"institution": "TeachReo"}, "geometry": {"type": "Point", "coordinates": [7.2685, 43.6989]}},
     {"type": "Feature", "properties": {"institution": "Aukfood"}, "geometry": {"type": "Point", "coordinates": [-1.4352446, 46.6741615 ]}},
     {"type": "Feature", "properties": {"institution": "TU Delft, Faculty TBM"}, "geometry": {"type": "Point", "coordinates": [4.3694, 52.0025459 ]}}
+	{"type": "Feature", "properties": {"institution": "RIADVICE"}, "geometry": {"type": "Point", "coordinates": [36.7088573,  10.2057799]}}
```

Comment:
I guess the coordinates were in the wrong order (and the indentation was different from the other lines):
```suggestion
    {"type": "Feature", "properties": {"institution": "RIADVICE"}, "geometry": {"type": "Point", "coordinates": [10.2057799, 36.7088573]}}
```